app/app.router.js

```diff
@@ -0,0 +1,23 @@
+var myApp = angular.module('myApp');
+myApp.run($trace => $trace.enable());
```

Comment:
Remember to remove this line :cop:, then you can use the angular module getter directly to define the config block below